Vagrantfile

```diff
@@ -27,6 +27,6 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    sudo yum install -y git livecd-tools
+    sudo yum install -y git livecd-tools docker parted
```

Comment:
@gbraad usually parted is already part of core packages but I just added it to be in safer side. I didn't see any policy file for this package and I think dependent also doesn't have but that I added to be safer side if you think it's an extra package we can remove it.

```
$ rpm -ql parted 
/lib64/libparted-fs-resize.so.0
/lib64/libparted-fs-resize.so.0.0.1
/lib64/libparted.so.2
/lib64/libparted.so.2.0.1
/sbin/parted
/sbin/partprobe
/usr/share/doc/parted
/usr/share/doc/parted/AUTHORS
/usr/share/doc/parted/NEWS
/usr/share/doc/parted/README
/usr/share/doc/parted/THANKS
/usr/share/info/parted.info.gz
/usr/share/licenses/parted
/usr/share/licenses/parted/COPYING
/usr/share/locale/ca/LC_MESSAGES/parted.mo
/usr/share/locale/cs/LC_MESSAGES/parted.mo
/usr/share/locale/da/LC_MESSAGES/parted.mo
/usr/share/locale/de/LC_MESSAGES/parted.mo
/usr/share/locale/es/LC_MESSAGES/parted.mo
/usr/share/locale/fr/LC_MESSAGES/parted.mo
/usr/share/locale/gl/LC_MESSAGES/parted.mo
/usr/share/locale/id/LC_MESSAGES/parted.mo
/usr/share/locale/it/LC_MESSAGES/parted.mo
/usr/share/locale/ja/LC_MESSAGES/parted.mo
/usr/share/locale/nl/LC_MESSAGES/parted.mo
/usr/share/locale/nn/LC_MESSAGES/parted.mo
/usr/share/locale/pl/LC_MESSAGES/parted.mo
/usr/share/locale/pt/LC_MESSAGES/parted.mo
/usr/share/locale/pt_BR/LC_MESSAGES/parted.mo
/usr/share/locale/ro/LC_MESSAGES/parted.mo
/usr/share/locale/ru/LC_MESSAGES/parted.mo
/usr/share/locale/rw/LC_MESSAGES/parted.mo
/usr/share/locale/sk/LC_MESSAGES/parted.mo
/usr/share/locale/sl/LC_MESSAGES/parted.mo
/usr/share/locale/sr/LC_MESSAGES/parted.mo
/usr/share/locale/sv/LC_MESSAGES/parted.mo
/usr/share/locale/tr/LC_MESSAGES/parted.mo
/usr/share/locale/uk/LC_MESSAGES/parted.mo
/usr/share/locale/vi/LC_MESSAGES/parted.mo
/usr/share/locale/zh_CN/LC_MESSAGES/parted.mo
/usr/share/locale/zh_TW/LC_MESSAGES/parted.mo
/usr/share/man/man8/parted.8.gz
/usr/share/man/man8/partprobe.8.gz
```